blocks/to-do/to-do.js

```diff
@@ -63,6 +63,7 @@ export default async function decorate(block) {
   input.setAttribute('type', 'text');
   input.setAttribute('id', 'to-do__input');
   orderedList.classList.add('to-do__ordered-list');
+  orderedList.setAttribute('data-has-listener', 'false');
```

Comment:
if you don't add this attribute as `'false'`, then checking for the attribute would return a falsy value anyway. so you can refactor the event listener attaching function accordingly, like

```
if (!list.getAttribute('data-has-listener')) {
    list.addEventListener('click', deleteListItem);
    list.setAttribute('data-has-listener', 'true');
  }
```